src/utils/download_file.ts

```diff
@@ -45,6 +45,6 @@ export async function downloadFile(
 		return { pathToFile, contentType };
 	} catch (error) {
 		writer.close();
-		throw new Error(`Failed to download file from remote path ${url}: ${error.message}`);
+		throw new Error(`Failed to download file from remote path ${url}: ${(error as Error).message}`);
```

Comment:
Same comment about ternary type check and interpolation fallback.

Reply:
5e860af